packages/controllers/organization_detail.coffee

```diff
@@ -5,7 +5,67 @@ if Meteor.isClient
   Template.organizationDetail.helpers
     organization: ->
       Organizations.findOne(@organizationId)
+    userIsMember: ->
+      UserProfiles.findOne({
+        userId: Meteor.userId(),
+        memberOfOrgs: @organizationId
+      })
+    userIsAdmin: ->
+      organization = Organizations.findOne(Template.instance().data.organizationId)
+      organization.userIsAdmin(Meteor.userId())
```

Comment:
I still find it weird that userIsMember and userIsAdmin are so different. Why doesn't organization also have a userIsMember method?